EMML/1001-2000/EMML1763.xml

```diff
@@ -1022,9 +1022,8 @@
       <change who="PL" when="2017-11-30">Created minimal record with link from data sent from Daniel Gullo, HMML project.</change>
         <change who="MKr" when="2022-05-12">Completed description</change>
         <change who="MKr" when="2022-09-11">Corrected work ID in #12, #76</change>
-
         <change who="MKr" when="2022-10-19">Addes missing work IDs</change>
-
+        <change who="MKr" when="2024-09-18">Minor corrections</change>
```

Comment:
I think "Minor corrections" is an understatement in this case. Better: "Corrections"